client/post-editor/editor-drawer/index.jsx

```diff
@@ -261,6 +262,17 @@ const EditorDrawer = React.createClass( {
 		);
 	},
 
+	renderCopyPost: function() {
+		// TODO: replace with currentPostTypeSupports( 'copy' );
```

Comment:
The [REST API documentation](https://developer.wordpress.com/docs/api/) and [Console](https://developer.wordpress.com/docs/api/console/) may serve to be very useful for you. In this case, specifically the [`/sites/%s/post-types` endpoint](https://developer.wordpress.com/docs/api/1.1/get/sites/%24site/post-types/).

That said, as far as I can tell, `copy` is not something that's a post type support. If we wanted to force it to be treated as a support, I've started to build out a new selector [`postTypeSupports`](https://github.com/Automattic/wp-calypso/blob/69c39a108f90dcd6a2f1ad35183133f4cb2801af/client/state/post-types/selectors.js#L41-L80) to serve as a replacement for the `currentPostTypeSupports` function in this component. `publicize` falls under a similar issue as `copy`, where we override whatever the REST API claims to be the support value (though in that case, `publicize` is a true `supports` value for other post types).

This all being said, is there a reason we want to allow post copying only for post types?